### PR TITLE
[FSSDK-10841] chore: release 5.0.0

### PR DESCRIPTION
**Pull request by muzahidul-opti**
## Summary
- Prepare for release 5.0.0

## Test plan
- All tests should be passed

## Issues
- [FSSDK-10841](https://jira.sso.episerver.net/browse/FSSDK-10841)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Optimizely Android X SDK Changelog
 
+## 5.0.0
+November 21st, 2024
+
+### Functionality Enhancements
```
Comment (raju-opti) [inline CHANGELOG.md:6]: this should be "breaking change"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Optimizely Android X SDK Changelog
 
+## 5.0.0
+November 21st, 2024
```
Comment (raju-opti) [inline CHANGELOG.md:4]: please update the date

**Review by raju-opti — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (muzahidul-opti) [reply CHANGELOG.md]: Updated

**Review by raju-opti — APPROVED** [review]
LGTM

**Pull request merged**